tests/tensor_search/test_get_stats.py

```diff
@@ -35,4 +37,6 @@ def test_get_stats_non_empty(self):
                 auto_refresh=True, device="cpu"
             )
         )
-        assert tensor_search.get_stats(config=self.config, index_name=self.index_name_1)["numberOfDocuments"] == 3
+        index_stats = tensor_search.get_stats(config=self.config, index_name=self.index_name_1)
+        assert index_stats["numberOfDocuments"] == 3
+        assert len(index_stats["size"]) != 0
```

Comment:
I saw that the computed size of the documents uploaded in this test was changing on each run; so figured `len` check would be a good assertion here, and added unit-tests for the bytes to human readable format function [here](https://github.com/marqo-ai/marqo/pull/492/files#diff-0b49ac0c0c74195999ae7eaf2c731b60fa971c62ee29aa6679b88d6309529fa8R402)